### DISCUSSION TITLE
各位好

### DISCUSSION THREAD

**Discussion by kanmanhin** [share121/inter-knot#4567] [灌水]
有一说一，不吹不黑，我来简单说两句吧，这个1.1版本的绝区零新剧情几乎直接就可以拿来当做米哈游走下坡路的证据了。

首先就是在人物塑造上面，这次剧情可以说是及其的失败，我实在无法理解为什么这次治安局作为一个保护新艾利都和未名市的巨大组织你这么大规模的事件竟然一声不吭还要掩盖这件事，有超級無敵巨tm可愛的救世主蓬萊寺九霄大人在還用得著這麼偷偷摸摸嗎。你散兵委托狡兔屋去调查空之律者的律者核心时我就觉得不对劲了,且不说你散兵你一个人偶要用神之心的拿律者核心干什么，就这不要命的任务你狡兔屋也真敢接啊?作为领导的景元都不把关一下的吗?可以说是人设崩坏.

第二就是在进入零号空洞的时候旅行者是已经知道停云被瓦尔特杨超越掉包了的，就这废物黄毛还硬是要启动第二神之键，最后要不是法厄同带着迪卢克和杰帕德開著真阿拉哈托1號機發動阿拉哈托射線前来救援估计都要狡兔屋和天命就全寄在里面了，这波莫名其妙的坑害奥托的操作完完全全就是降智无脑的傻卵子操作。

第三就是新地图，新地图的设计疑似夹带私货，为什么要设置在在龙脊雪山的鳞渊镜下方，还要在贝洛伯格透過匹诺康尼的特制崩坏能世界泡以及要依靠知更鸟的协乐帮助才能闯进去。在这下面打空之律者合理怀疑这就是米哈游想要断掉仙舟龙脉，这隐喻什么大家都知道了吧，这其心可诛，我的一个道姑朋友用了占卜術看了直呼恶心

第四为什么每次的任务都是去未名市每周做好人好事?什么楼上装修太吵的，买周边插队的，帮忙挂装饰品的，每一周真的有那么多人要帮忙吗？你就拿这个推进剧情啊?明明反主都知道了这个世界的崩坏就是nxx，但是和蔷薇和4位男主谁都不说，最后夏彦意外从老杨手上拿到了律者核心才解决了这次危机，左然拿到了水系的神之眼浇灭了边上引燃的建筑，莫弈直接用心理研究所的机械救下了旁边的市民，而陆景和组织了全公司的人去救援，女主还念叨了什么旅行者、开拓者和绳匠，真是什么莫名其妙的剧情，褒姒吧

总之这次1.1主线实在让人非常失望，知名游戏测评人【欢愉】啊哈直接给游戏打出了2.5星的差评，要不是有米哈游滤镜可能还会更低。大家快点退坑吧。我在这里10块低价收购失望玩家的账号，反正mhy的账号很快就要贬值了我来收購吧。
。
。
。
。
。
。
。
。
。
。
。
。
。
。
奇书我是沙币

**Comment by Dave-12138** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/03/28/ca2049b51a279525b81ef811cf505b68_6579008794204639205.png" alt="加拉赫 嗯？" width="75" >什么构史

**Comment by Signedinn** [comment]
虚构史学家打来了？
![IMG_4427](https://github.com/user-attachments/assets/38d85a87-a251-40b3-9d0d-95b39bfe02f4)

**Comment by KELERSAMA** [comment]
简直就是嬴政摸电线！米游玩家终于觉醒了，放心，米哈游会出手的！剧情里这个黄毛旅行者明明知道边上的白色飞行物是王小美，却依然要去勾搭旁边的绝区零的星，不知道居心何在，怎么看剧情怎么有问题。最重要的是星穹铁道里的治安局，他们为什么要去阻止盗洞客成为开拓者前往提瓦特大陆进行旅行？太可怕了。
